src/KygekDev/IsOP/IsOP.php

```diff
@@ -25,18 +25,19 @@ class IsOP extends PluginBase {
     private const COMMAND_PERMISSION = "isop.command";
 
     public function onCommand(CommandSender $sender, Command $command, string $label, array $args) : bool {
-        if ($command->getName() !== "isop" || !$sender->hasPermission(self::COMMAND_PERMISSION)) return true;
+        if ($command->getName() !== "isop" or !$sender->hasPermission(self::COMMAND_PERMISSION)) return true;
         
         if (isset($args[0])) {
-            if (in_array(strtolower($args[0]), array_map("strtolower", $this->getOpList()))) {
+            if (in_array(mb_strtolower($args[0]), array_map("strtolower", $this->getOpList()))) {
                 $this->sendMessage($sender, "Player " . $args[0] . " is OP");
             } else {
                 $this->sendMessage($sender, "Player " . $args[0] . " is not OP");
             }
             return true;
         }
-        
-        $this->sendMessage($sender, "OP players: " . implode(", ", $this->getOpList()));
+        $getOpList = implode(", ", $this->getOpList());
+        $opList = ($getOpList == null) ? TF::RED . "Null" :  $getOpList;
```

Comment:
1. Using `empty()` is better than `== null` because $getOpList returns `""` if $this->getOpList() returns `[]`.
2. Changed to `-` because for me it doesn't make sense to print `OP players: Null` (And what if a player named `Null` is OP-ed?)

```suggestion
        $getOpList = implode(", ", $this->getOpList());
        $opList = (empty($getOpList)) ? "-" :  $getOpList;
```